SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -41,25 +53,42 @@ Source303:      test_RECORD
 Source901:      README.md
 Source902:      LICENSE
 
-%if 0%{?with_check}
-BuildRequires:  python3-atomicwrites
-BuildRequires:  python3-attrs
-BuildRequires:  python3-pip
-BuildRequires:  python3-pluggy
-BuildRequires:  python3-six
+%if %{with tests}
+BuildRequires:  python3dist(pytest)
+%if %{with pytest_xdist}
+BuildRequires:  python3dist(pytest-xdist)
+%endif
+BuildRequires:  python3dist(pyyaml)
 BuildRequires:  python3dist(packaging)
 BuildRequires:  python3dist(pip)
-BuildRequires:  python3dist(pyyaml)
 BuildRequires:  python3dist(setuptools)
+%if %{with tox_tests}
 BuildRequires:  python3dist(tox-current-env) >= 0.0.6
+%endif
 BuildRequires:  python3dist(wheel)
+#BuildRequires:  (python3dist(tomli) if python3 < 3.11)
```

Comment:
Can probably be removed. We don't have anything below Python 3.12 anyway.

Reply:
removed